### ISSUE TITLE
Return type error with useTweaks

### ISSUE THREAD

**Issue by lourd** [tweakpane/use-tweaks#43]
When using the `useTweaks` hook, I'm getting an error about the return types. The return type looks to be an object with a single key that is an empty string?

```ts
const {
    numCirclesPerGroup,
    circleGap,
    innerCircleRadius,
    numGroups,
  } = useTweaks({
    numCirclesPerGroup: { value: 12, min: 1, max: 40, step: 1 },
    circleGap: { value: 2, min: 0, max: 100, step: 1 },
    innerCircleRadius: { value: 20, min: 1, max: 100, step: 1 },
    numGroups: { value: 6, min: 2, max: 90, step: 1 },
  })
// Property 'innerCircleRadius' does not exist on type '{ "": { numCirclesPerGroup: ....
```

See this sandbox https://codesandbox.io/s/jovial-lamport-8c4u3?file=/src/App.tsx

**Comment by unphased** [comment]
Pretty sure now that if you revert your project to react 16 this will go away.

**Comment by cocopon** [comment]
Fixed in 0.3.2-alpha.0.
